dev/com.ibm.ws.ejbcontainer.security/src/com/ibm/ws/ejbcontainer/security/internal/EJBSecurityCollaboratorImpl.java

```diff
@@ -399,19 +399,14 @@ public void populateAuditEJBHashMap(EJBRequestData request) {
         String methodInterface = methodMetaData.getEJBMethodInterface().specName();
         String methodSignature = methodMetaData.getMethodSignature();
         String beanName = methodMetaData.getEJBComponentMetaData().getJ2EEName().getComponent();
-        List<Object> methodParameters = null;
-        if (methodArguments != null && methodArguments.length > 0) {
-            methodParameters = Arrays.asList(methodArguments);
-        }
 
         ejbAuditHashMap.put("applicationName", applicationName);
         ejbAuditHashMap.put("moduleName", moduleName);
         ejbAuditHashMap.put("methodName", methodName);
         ejbAuditHashMap.put("methodInterface", methodInterface);
         ejbAuditHashMap.put("methodSignature", methodSignature);
         ejbAuditHashMap.put("beanName", beanName);
-        ejbAuditHashMap.put("methodParameters", methodParameters == null ? "null" : methodParameters.toString());
-
+        ejbAuditHashMap.put("methodParameters", ParameterUtils.format(methodArguments).toString());
```

Comment:
The output for null is changed.  Before it returned "null".  Now it will just be empty String.  If you are trying not to change the format, you probably should still display "null" when the methodParameters are null.

Reply:
That's true. I missed that part. thanks.